src/FSCache.js

```diff
@@ -30,16 +31,31 @@ class FSCache {
     return path.join(this.dir, hash + '.json');
   }
 
+  async getLastModified(filename) {
+    let mtime = 0;
+    if (isGlob(filename)) {
+      let files = await glob(filename, {
+        strict: true,
+        nodir: true
+      });
+      for (let file of files) {
+        let fileStats = await fs.stat(file);
+        if (fileStats.mtime > mtime) {
+          mtime = fileStats.mtime.getTime();
+        }
+      }
+    } else {
```

Comment:
nit-pick: use `Promise.all` for better perf:
```js
if (isGlob(filename)) {
    let files = await glob(filename, {
        strict: true,
        nodir: true
    });

    mtime = (
        await Promise.all(
            files.map(file =>
                fs.stat(file).then(stat => stat.mtime.getTime())
            )
        )
    ).reduce((a, b) => Math.max(a, b), 0)
}
```

Reply:
I'll change it